frontend/express/views/dashboard.html

```diff
@@ -2501,7 +2504,9 @@ <h4>{{this.name}}</h4>
 		 Countly.userData.set("lastVersion", "<%= countlyVersion %>");
 		 Countly.userData.set("lastEdition", "<%= countlyTypeTrack %>");
 		 Countly.userData.set("isTrial", <%= countlyTrial %>);
-		 Countly.userData.set("cpus", <%= cpus %>);
+         <% if (!offline_mode) { %>
+            Countly.userData.set("cpus", <%= cpus %>);
+         <% } %>
 		 Countly.userData.push_unique("servers", window.location.hostname);
 		 Countly.userData.push_unique("versions", "<%= countlyVersion %>");
 		 Countly.userData.push_unique("editions", "<%= countlyTypeTrack %>");
```

Comment:
You should also disable intercom that goes further in this file
And recheck all use_google flags, I think there was something more.